packages/sidebar-settings/src/blocks/dropdown.ts

```diff
@@ -24,6 +24,11 @@ export type DropdownBlock<AppBridge> = {
      */
     size?: 'small' | 'large' | DropdownSize;
 
+    /**
+     * Whether the dropdown should open to top or to bottom.
+     */
+    position?: 'top' | 'bottom';
```

Comment:
The devs shouldn't be able to control that, it should be automatic in the sidebar

Reply:
![image](https://github.com/Frontify/brand-sdk/assets/30796791/fac2bea4-cc0c-4e70-aeec-5b96967aba0e)

For me it always drops down even if there is limited space. The height of the dropdown area is dynamic, but the position seems to be hardcoded to bottom.

Reply:
We can go for an approach where we tweak the component in Fondue to automatically open to a position where there is more space.

Reply:
Should be fixed within the sidebar instead of the settings (maybe the Fondue Dropdown already support the `flip` option?)

Reply:
I will close this PR and fix it in Fondue.